pkgs/by-name/fi/firefox-sync-client/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "firefox-sync-client";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "Mikescher";
+    repo = "firefox-sync-client";
+    rev = "v${version}";
+    sha256 = "sha256-Ax+v4a8bVuym1bp9dliXX85PXJk2Qlik3ME+adGiL1s=";
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
    hash = "sha256-Ax+v4a8bVuym1bp9dliXX85PXJk2Qlik3ME+adGiL1s=";
```

Reply:
Ah! Should have thought about updating to `hash` before uploading to be fair.

I thought the idea was to keep using `rev = <tag>` and only use `refs/tags/<tag>` for upstreams that do clashing branch names.

Reply:
(`rev = <tag>` is what's used in the documentation, never `rev = refs/tags/<tag>`. So will keep it as-is).

Reply:
> (rev = <tag> is what's used in the documentation, never rev = refs/tags/<tag>. So will keep it as-is).

This doesn't mean it's correct. See https://github.com/NixOS/nixpkgs/issues/162454

It is also being [widely adopted](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20%22refs%2Ftags%22&type=code) across the tree